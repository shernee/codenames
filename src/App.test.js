import React from 'react';
import { render, cleanup } from '@testing-library/react';
import App from './Trivia';

afterEach(cleanup)

test ('take a snapshot', () => {
  const { asFragment } = render(<App/>)

  expect(asFragment(<App/>)).toMatchSnapshot()
});