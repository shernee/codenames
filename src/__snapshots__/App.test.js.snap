// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`take a snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="main-container"
    >
      <div
        style="width: 40%; text-align: center;"
      >
        <div
          style="color: black;"
        >
          <h1>
            Indias capital is
          </h1>
        </div>
        <div
          class="btn-group"
        >
          <button>
            Kolkata
          </button>
        </div>
        <div
          class="btn-group"
        >
          <button>
            New Delhi
          </button>
        </div>
        <div
          class="btn-group"
        >
          <button>
            Bangalore
          </button>
        </div>
        <div
          class="btn-group"
        >
          <button>
            Mumbai
          </button>
        </div>
      </div>
      <div>
        <h2
          style="color: green;"
        >
          Correct: 0
        </h2>
        <h2
          style="color: red;"
        >
          Incorrect: 0
        </h2>
      </div>
    </div>
    <br />
  </div>
</DocumentFragment>
`;
